packages/components/inputs/search-select-input/src/search-select-input.tsx

```diff
@@ -258,6 +258,10 @@ export type TSearchSelectInputProps = {
    * The style of the an option in the dropdown menu. It could be single lined option or an option with more and custom info
    */
   optionType?: 'single-property' | 'double-property' | 'multiple-properties';
+  /**
+   * Icon to display on the left of the placeholder text and selected value. Has no effect when `isMulti` is enabled.
```

Comment:
Is the `Has no effect when `isMulti` is enabled.` expected? 🤔

```suggestion
   * Icon to display on the left of the placeholder text and selected value.
```

Reply:
@emmenko I would expect so, since all other `selectInputs` and `selectFields` behave the same way. So if the input has set `isMulti` to `true`, then we hide the `leftIcon`.

Reply:
oh yeah good catch, let's follow the same approach as we do in other select inputs

Reply:
Alright, thanks of the clarification 🤗